### PR TITLE
Remove `wee_alloc` in favor of `rlsf`

### LINKED ISSUE #401: wee_alloc: wasm memory leaks for large allocations
Find an alternative to wee_alloc or use the default.

The crate is not maintained:
https://github.com/rustwasm/wee_alloc/issues/107

And there’re possible leaks for large allocations:
https://github.com/rustwasm/wee_alloc/issues/106

### LINKED ISSUE #808: switch wasm allocator
we're currently using wee_alloc, which is not maintained. One option for replacement is https://github.com/yvt/rlsf (thanks @sug0 for the suggestion)

### PR DESCRIPTION
**Pull request by Fraccaman**
## Describe your changes
Closes #808. 
Closes #401.

## Indicate on which release or other PRs this topic is based on

0.37.0

## Checklist before merging to `draft`
- [x] I have added a changelog
- [x] Git history is in acceptable state

### CODE REVIEW

File: genesis/localnet/parameters.toml [hunk 1]
```diff
@@ -108,4 +108,4 @@ stewards_inflation_rate = "0.01"
 # default mint limit of each token
 default_mint_limit = "0"
 # default per-epoch throughput limit of each token
-default_per_epoch_throughput_limit = "0"
+default_per_epoch_throughput_limit = "0"
```
Comment (tzemanovic) [inline genesis/localnet/parameters.toml:111]: nit: skip this change

**Review by tzemanovic — APPROVED** [review]

File: genesis/localnet/parameters.toml (hunk 1 above)
Comment (sug0) [reply genesis/localnet/parameters.toml:111]: +1

**Review by sug0 — DISMISSED** [review]
lgtm. just a nit (same as tomas')

**Review by tzemanovic — APPROVED** [review]

File: wasm_for_tests/wasm_source/Cargo.toml
Comment (tzemanovic) [inline wasm_for_tests/wasm_source/Cargo.toml:1]: oh looks like this file was accidentally revived at some point

**Pull request merged**